test/mysql.test.js

```diff
@@ -912,6 +922,92 @@ describe('mysql', function() {
     });
   });
 
+  context('match operator', function() {
```

Comment:
let's add another test for the new suggested operator `matchnlqe` or `nlqematch`.

Reply:
Done